src/components/app/details/cicdHistory/LogsRenderer.tsx

```diff
@@ -36,8 +39,13 @@ export default function LogsRenderer({
         renderConfigurationError(isBlobStorageConfigured)
     ) : (
         <div className="logs__body">
-            {logs.map((log, index) => {
-                return <p className="mono fs-14" key={`logs-${index}`} dangerouslySetInnerHTML={createMarkup(log)} />
+            {logs.map((log: string, index: number) => {
+                return (
+                    <div className="flex left" key={`logs-${index}`}>
+                        <span className="cn-4 dc__mb-auto col-2 pr-10 mt-1">{index + 1}</span>
```

Comment:
Do we need mt-1?

Reply:
I believe it's required because at first I've tried it with out ```mt-1``` and it's not aligned with the logs that way. So as you can see in the above shared screenshots it's properly aligned afterwards.